tests/Unit/IO/Observers/Test_RegisterElements.cpp

```diff
@@ -104,6 +115,7 @@ void check_observer_registration() {
     runner.simple_action<
         element_comp,
         observers::Actions::RegisterWithObservers<TypeOfObservation>>(id, 0);
+    runner.invoke_queued_simple_action<obs_component>(0);
```

Comment:
The documentation for `simple_action` says the action should run immediately (and the code appears to back that up), so why is `invoke_queued_simple_action` necessary?

Reply:
Ah, I think a comment on this line stating which action this invokes would be useful, no?